components/create/aliases.tsx

```diff
@@ -93,7 +93,8 @@ function Aliases() {
           return err;
         });
 
-        if (isNaN(Number(values.effectivePeriod))) {
+        const parsedNumber = Number(values.effectivePeriod);
+        if (isNaN(parsedNumber) || parsedNumber <= 0) {
```

Comment:
Sorry, not equal `0`. it doesn't make sense to have 0 duration and contract part will throw error for 0 duration.